app/core/models.py

```diff
@@ -0,0 +1,31 @@
+from datetime import date
+
+from sqlmodel import Field
+from sqlmodel import SQLModel
+
+
+class ExpenseCategoryGroup(SQLModel, table=True):
+    id: int | None = Field(default=None, primary_key=True)
```

Comment:
better use Optional, its a union of x and None

Reply:
So yea this what I saw being used in that template. I thought "Optional" was more the old Python type-hint style, no?

Reply:
Both are fine. i guess that you are right here